addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -303,6 +303,12 @@
 		
 		"Engineer"
 		{
+			"Secondary"
+			{
+				"desp"		"Secondary (Pistols): {positive} Increased accuracy and reload speed"
+				"attrib"	"106 ; 0.50 ; 97 ; 0.30"
```

Comment:
I'm not sure if you intend 30% or 70% faster reload time but `0.30` would be 70%.
Although I wonder if pistol could instead have no reload necessary.